native_client/definitions.mk

```diff
@@ -78,7 +78,7 @@ PYVER := $(shell python -c "import platform; maj, min, _ = platform.python_versi
 PYTHON_PACKAGES      :=
 PYTHON_PATH          := PYTHONPATH=$(RASPBIAN)/usr/lib/python$(PYVER)/:$(RASPBIAN)/usr/lib/python3/dist-packages/
 NUMPY_INCLUDE        := NUMPY_INCLUDE=$(RASPBIAN)/usr/include/python3.7m/
-PYTHON_SYSCONFIGDATA := _PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_m_linux_arm-linux-gnueabihf
+PYTHON_SYSCONFIGDATA := $(shell find /usr/lib/python3.*/ -type f -name "*.py" | grep sysconfigdata_)
```

Comment:
same

Reply:
What you think we should be doing instead?

Reply:
this was likely a suggestion to run manually and find the right file name you needed there.

Keep `_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_m_linux_arm-linux-gnueabihf` but use the correct value, based on what `$(shell find /usr/lib/python3.*/ -type f -name "*.py" | grep sysconfigdata_)` returns

Reply:
perfect trying now :)

Reply:
Yes my suggestion was to use the find command to locate the module manually. We thought about making it dynamic but it add complexity (specifically with the multibyte thingy).